src/components/ScheduleConfirmationModal/hooks/useScheduleConfirmation.ts

```diff
@@ -0,0 +1,100 @@
+import React, { useEffect, useState } from 'react';
+import useGetSchedulesEndingRequest from '../../../service/requests/useGetSchedulesEndingRequest';
+import { TScheduleEnding } from '../../../service/requests/useGetSchedulesEndingRequest/types';
+import useUpdateScheduleEnding from '../../../service/requests/useUpdateScheduleEnding';
+import { useSnackBar } from '../../../utils/renderSnackBar';
+
+const useScheduleConfirmation = () => {
+  const [isOpen, setIsOpen] = useState(false);
+  const { showErrorSnackBar, showSuccessSnackBar, showInfoSnackBar } =
```

Comment:
Remove esse parametro já que vc não usa. Tá dando warning no console.
```suggestion
  const { showErrorSnackBar, showInfoSnackBar } =
```